subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -15,7 +15,19 @@
 /* Link Layer header size of Adv PDU. Assumes pdu_adv is packed */
 #define PDU_AC_LL_HEADER_SIZE  (offsetof(struct pdu_adv, payload))
 /* Advertisement channel maximum PDU size */
-#define PDU_AC_SIZE_MAX        (PDU_AC_LL_HEADER_SIZE + PDU_AC_PAYLOAD_SIZE_MAX)
+#define PDU_AC_SIZE_MAX	(PDU_AC_LL_HEADER_SIZE + PDU_AC_PAYLOAD_SIZE_MAX)
+
+#define ACCESS_ADDR_SIZE        4
+#define ADVA_SIZE               6
+#define SCANA_SIZE              6
+#define INITA_SIZE              6
+#define TARGETA_SIZE            6
+#define LLDATA_SIZE             22
+#define CRC_SIZE                3
+#define PREAMBLE_SIZE(phy)      (phy&0x3)
+#define LL_HEADER_SIZE(phy)     (PREAMBLE_SIZE(phy) + PDU_AC_LL_HEADER_SIZE \
+				  + ACCESS_ADDR_SIZE + CRC_SIZE)
+#define BYTES2US(bytes, phy)    ((bytes)*8/BIT((phy&0x1)>>1))
```

Comment:
nit: We should use `PKT_US` only.
`PKT_US` is now available for all to use (was limited to Nordic at first), which turns `BYTES2US` into duplicate code.

Reply:
PKT_US() macro is local to ull_conn.c and could possibly be modified to utilize BYTES2US.